tests/characters/test_alice_can_grant_and_revoke.py

```diff
@@ -96,6 +102,37 @@ def test_federated_grant(federated_alice, federated_bob):
         assert kfrag == retrieved_kfrag
 
 
+@pytest.mark.usefixtures('federated_ursulas')
+def test_revocation(federated_alice, federated_bob):
```

Comment:
Do you want this test cover blockchain I/O or is this aimed at federated only?

Reply:
This PR is aimed at Federated only. Definitely need to discuss more on blockchain revocation.

Reply:
Needs an Issue.

Reply:
See #570